.github/workflows/pr.yml

```diff
@@ -5,59 +5,79 @@ on:
     branches: [ main ]
 
 env:
-  CARGO_TERM_COLOR: always
+  CARGO_INCREMENTAL: 1
 
 jobs:
-  test-and-lint:
+  test-linux:
```

Comment:
![image](https://user-images.githubusercontent.com/3018448/234201941-dc3818e0-381b-4e61-8fd7-bd1ed7328046.png)

Doc: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs
Example: https://github.com/cecton/mbrman/blob/main/.github/workflows/rust.yml